packages/@aws-cdk/aws-backup/README.md

```diff
@@ -63,6 +63,15 @@ plan.addRule(new backup.BackupPlanRule({
 }));
 ```
 
+Continuous backup and point-in-time restores (PITR) can be configured:
```

Comment:
Please mention requirements for `deleteAfter` and `moveToColdStorageAfter` here as well.